### ISSUE TITLE
Camera

### ISSUE THREAD

**Issue by anantafp** [keaneflynn/yolo-MeasureObjects#3]
is there any difference if i use intel realsense camera/ webcam camera?
look forward for your answer

**Comment by keaneflynn** [comment]
Yes, this program will only work with an intel realsense camera. It is not possible to accurately measure object in an image with a single camera, they must be stereo cameras. Which the intel realsense camera is. Cheers

**Comment by anantafp** [comment]
okay, thank you for your response
